api-server/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@freecodecamp/api-server",
-  "version": "0.0.1",
+  "version": "1",
```

Comment:
```suggestion
  "version": "1.0.0",
```
🙃

Reply:
You understand this will break it now...

Reply:
We should fix it then 😛

Reply:
🤦‍♂️